fcrepo-api-x-model/src/main/resources/ontologies/apix-extension.ttl

```diff
@@ -0,0 +1,65 @@
+@prefix apix:<http://fedora.info/definitions/v4/api-extension#> .
+@prefix fcr:<http://fedora.info/definitions/v4/repository#> .
+@prefix owl:<http://www.w3.org/2002/07/owl#> .
+@prefix rdfs:<http://www.w3.org/2000/01/rdf-schema#> .
+@prefix svc:<http://fedora.info/definitions/v4/service#> .
+@prefix xsd:<http://www.w3.org/2001/XMLSchema#> .
+
+apix: a owl:Ontology;
+    owl:imports svc:;
+    rdfs:label "API Extension ontology" .
+
+apix:Extension a owl:Class;
+    rdfs:label "API-X extension"@en;
+    rdfs:comment
+    "Relates Services to Repository objects. May expose a service on repository resources (‘exposing’ modality), or modify request or responses to repository objects (‘intercepting’ modality)."@en .
```

Comment:
Ha, That was a cut 'n paste from the rendered markdown - I didn't realize it did that.